hlx_statics/components/link.js

```diff
@@ -0,0 +1,6 @@
+export default function decorateLink(link, variant = 'primary') {
```

Comment:
EDS equivalent of Gatsby [Link](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Link/index.js#L18) component